### ISSUE TITLE
crazy idea!!

### ISSUE THREAD

**Issue by blackdragonIV** [sinkillerj/ProjectE#969]
i will say that there is a chance of 98% that someone will punch me in the face (plz dont) and tell me this is impossible but i will post it anyway for the lolz

this idea may be buggy, crashy, and possibly will require alot of looking into the API of other mods

simply my idea is the ability to STOP time using watch of flowing time

stopping time will include effects that will occur in the player rendered chunks including

1 no machine ticking ; simply you can still look into GUI and move items from chests/ furnaces but furnaces and other mods machines stops working and progress bars will be frozen in place also collectors wont work and spawners will not spawn also machines will not use liquids/fuel/energy/whatever to do their job... because reasons

2 no day/night ; the sun/moon will be frozen in place

3 no mobs spawning and mobs frozen; mobs that have already spawned will be frozen in place and can not move. skeletons will not be able to shoot arrows , creepers will not explode if near them and zombies can not or can harm you only if you are close to them (since they cant move anyway) and also you can hit them, they will take damage but no knockback, also traveling to a new chunk will not spawn new mobs until the ability is turned off. pretty much like /freeze command which existed in the mod single players command back in mc 1.0 and lower but with some evil touches >:D

4 no animations/ particles moving (mostly no possible) ; will freeze animation such as smoke from torches and mobs spinning sprites inside of mob spawners

5 no redstone/ energy flow ; redstone will not move in wires and stop at which line it was in the moment the ability was activated also energy like RF/EU/mana (it would be very interesting to see mana burst just frozen in the air :0) will not travel also it may freeze timers and cells from project red/ blue power also things such as water /lava will not flow and fire wont spread

6 items and arrows ; arrows will fly very slow but will remain in the same trajectory also dropped items will fall very slowly and possibly have a longer despawn time?

of course please add config to disable this functionality because you can imagine how much abuse on servers this will get XD

yes mostly it has more draw backs than fun to play with stuff but people may use it with redstone contraptions also i just want to watch the world burn :D

**Comment by Roblomal** [comment]
Haha lol

Date: Mon, 24 Aug 2015 07:57:35 -0700

**Comment by williewillus** [comment]
this will break shit so badly I don't want to imagine how other modders will react

tldr no

**Comment by blackdragonIV** [comment]
can it at least be a plugin for the mod in the future?

**Comment by williewillus** [comment]
This kind of thing can't be a plugin, it would just be its own mod.
The mod could be "ProjectE themed" of course but it wouldn't actually interact with PE in any way

**Comment by blackdragonIV** [comment]
will a plugin/mod that adds a mk2/ advanced watch of flowing time that is designed for people to watch the world burn :D?

**Comment by Carbon-Ghost** [comment]
I think RF tools did that, but it's inworld only lol. But this is madness ;O

**Comment by blackdragonIV** [comment]
no this is SPARTA
